docs/src/pages/languages/python.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Python
+menu: Languages
+edit: false
+---
```

Comment:
Currently, this page is being served at `/src-pages-languages-python` generated by docz as a default based on the filepath in the source. To make things clean, you can specify a defined path (perhaps `/langages-python` in this case) by adding a `route` attribute in the document settings. See [this doc](https://www.docz.site/docs/document-settings) for more info.

[Here's](https://raw.githubusercontent.com/approzium/approzium/develop/docs/src/pages/quickstart_.mdx) an example.

Reply:
Good catch! Addressed it now.